evaluators/rank/fScoreEvaluator/README.md

```diff
@@ -0,0 +1,73 @@
+# fScoreEvaluator
```

Comment:
What about following the camel case here? The class can be named as `FScoreEvaluator` as well.
```suggestion
# FScoreEvaluator
```